handlers/health.go

```diff
@@ -0,0 +1,204 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/kiali/kiali/services/business"
+)
+
+const defaultHealthRateInterval = "10m"
+
+// NamespaceHealth is the API handler to get app-based health of every services in the given namespace
```

Comment:
The params used by this new handler are documented ?
This was the first file of the PR, not yet reviewed all the code.

Reply:
Ok, I see the struct used is annotated with swagger.